index.src.html

```diff
@@ -2033,6 +2033,8 @@ <h3 id="user-mediated-selection">Credential Selection</h3>
   The chooser interface SHOULD include all {{Credential}} objects associated
   with the origin that requested credentials.
 
+  The chooser interface MAY offer the user to enter and save new credentials.
```

Comment:
How about something like "The chooser interface MAY offer the user the ability to create new credentials directly. For example, the user agent could provide a form which accepted a username and password, and returned a newly-created `PasswordCredential` containing that information."

It might also be worthwhile to capture @mnoorenberghe's suggestions in #18 in an "Implementation Considerations" section to make it clear why such an interface might be valuable.